src/main/java/core/basesyntax/service/impl/ReadFromCsv.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CustomFileReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReadFromCsv implements CustomFileReader {
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public List<FruitTransaction> readFile(String filePath) {
```

Comment:
this method violates SRP, you should read from the file here, for conversion to FruitTransaction you should create a separate service